app/ide-desktop/lib/client/esbuild-config.ts

```diff
@@ -55,9 +57,13 @@ export function bundlerOptions(
         define: {
             BUNDLED_ENGINE_VERSION: JSON.stringify(bundledEngineVersion),
             PROJECT_MANAGER_IN_BUNDLE_PATH: JSON.stringify(projectManagerInBundlePath),
+            'process.env.ELECTRON_DEV_MODE': JSON.stringify(String(devMode)),
+            'process.env.GUI2_CONFIG_PATH': JSON.stringify(
```

Comment:
Should this env var be renamed to `GUI_CONFIG_PATH` now that `GUI2` is default?

Reply:
yeah - nice catch